_layouts/page-with-notification.html

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+---
+
+{%- capture notification %}
+Testing
```

Comment:
Could this be a value that's defined in the config file?

Reply:
@mlubin sure! Will shift it over